src/main/java/core/basesyntax/lotteryball/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.lotteryball;
+
+import java.util.Random;
+
+public class Lottery {
+    private Ball ball;
```

Comment:
I dont think this variable should be class level. Lets make it local (create and use only in getRandomBall() method)